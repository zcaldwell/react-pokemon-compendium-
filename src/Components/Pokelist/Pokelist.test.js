import { render } from '@testing-library/react';
import PokeList from './Pokelist';

test('renders pokemon list', () => {
  const container = render(<PokeList pokemon={['pikachu']} />);
  expect(container).toMatchSnapshot();
});
