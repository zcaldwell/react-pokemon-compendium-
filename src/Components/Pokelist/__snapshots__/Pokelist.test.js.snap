// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders pokemon list 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="card-container"
      >
        <div
          class="pokecard"
        >
          <h2>
             Name: 
             
          </h2>
          <img />
          <div
            class="stats"
          >
            <h3>
               Type: 
               
            </h3>
            <h3>
               Speed: 
               
            </h3>
            <h3>
               Attack: 
               
            </h3>
            <h3>
               Defense: 
               
            </h3>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="card-container"
    >
      <div
        class="pokecard"
      >
        <h2>
           Name: 
           
        </h2>
        <img />
        <div
          class="stats"
        >
          <h3>
             Type: 
             
          </h3>
          <h3>
             Speed: 
             
          </h3>
          <h3>
             Attack: 
             
          </h3>
          <h3>
             Defense: 
             
          </h3>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
